### PR TITLE
Create fy25-dart-budget-proposal-response.md

### PR DESCRIPTION
**Pull request by LEQUI000**
Create draft letter for cta to email dart board 

Due date ASAP

### CODE REVIEW

File: _posts/fy25-dart-budget-proposal-response.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+author:
+  name: DATA Alert
+  picture: "/assets/blog/authors/data-team.png"
+ogImage:
+  url: "/assets/blogfy25-dart-budget-proposal-response/cover.jpg"
+banner: 
+  text: "ACTION REQUESTED: EMAIL YOUR DART BOARD ABOUT THE UPCOMING FY2025 DART BUDGET PROPOSALS"
+---
+
+DATA's August newsletter highlights key events, including our General Meeting on August 18 and upcoming public meetings at the DART Board and Dallas City Council.
```
Comment (CurtisGreen) [inline _posts/fy25-dart-budget-proposal-response.md]: This is copy-pasted from the previous article, need to update or remove it

File: _posts/fy25-dart-budget-proposal-response.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+## Contact your DART board members!
+https://dart.org/about/about-dart/board-members
```
Comment (Schobbish) [inline _posts/fy25-dart-budget-proposal-response.md]: raw link doesn't work

File: _posts/fy25-dart-budget-proposal-response.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+---
+
+DATA's August newsletter highlights key events, including our General Meeting on August 18 and upcoming public meetings at the DART Board and Dallas City Council.
+
+## Contact your DART board members!
+https://dart.org/about/about-dart/board-members  
+
+DART board contact list:
+```
+GASlagel@dart.org , RSchlosser@dart.org, MNBarbera@dart.org, FMHernandez@dart.org, MCEnoch@dart.org, PNWageman@dart.org, RHStopfer@dart.org, MWKrause@dart.org, PJKennedy@dart.org, DHrbacek@dart.org, CGarcia3@dart.org, MAbraham@dart.org, EMacGregor@dart.org, DDAlexander@dart.org , RBBryant@dart.org
```
Comment (Schobbish) [inline _posts/fy25-dart-budget-proposal-response.md]: a couple commas have spaces before

File: _posts/fy25-dart-budget-proposal-response.md (hunk 1 above)
Comment (Schobbish) [inline _posts/fy25-dart-budget-proposal-response.md:15]: this supposed to be kept?

File: _posts/fy25-dart-budget-proposal-response.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+Thank You,
+
+**[Your Name]**
+
+
+DART board contact list:
+```
+GASlagel@dart.org, RSchlosser@dart.org, MNBarbera@dart.org, FMHernandez@dart.org, MCEnoch@dart.org, PNWageman@dart.org, RHStopfer@dart.org, MWKrause@dart.org, PJKennedy@dart.org, DHrbacek@dart.org, CGarcia3@dart.org, MAbraham@dart.org, EMacGregor@dart.org, DDAlexander@dart.org, RBBryant@dart.org
+```
```
Comment (CurtisGreen) [inline _posts/fy25-dart-budget-proposal-response.md]: Contact list is included twice, one of them should be removed

File: _posts/fy25-dart-budget-proposal-response.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+* Reductions in expansion of Armed Security guard contract, which will eliminate expanded security coverage at employee facilities, Silver Line, or clean teams
+  * This may concern Blake Margolis, mayor of Rowlett. He presided over a resolution to cut DART’s funding in part due to safety and fare enforcement concerns
+* Light-Rail frequency improvements on weekends will be halted
+  * At least that means Farmers Branch City Councilor Richard Jackson will not have to worry about as much trash coming into Farmers Branch on DART trains. 
+* Reversal of new and improved bus routes and service included in FY2024 budget
+* Non-performing routes at risk of reduction or elimination: 45 Marsalis (reduction), 234 Parker, 245 Shiloh, Lakewood GoLink zone
+* High passenger subsidy routes to be elimintated: 305 Addison Express, 308 NW Plano Express, 378 Red Bird Express, 383 Lake Ray Hubbard Express
+  * Many of these routes are in Plano, which might trouble the city manager, Mike Israelsson, who wants to cut funding in part due to empty buses on one-hour intervals
```
Comment (Schobbish) [inline _posts/fy25-dart-budget-proposal-response.md]: in the original doc there were links to stuff on these bullet points

File: _posts/fy25-dart-budget-proposal-response.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+* Other routes to be eliminated: 402 NorthPark Shuttle, South Dallas GoLink, Inland Port GoLink
+
+The 0% Cap, which effectively means no increase in operating funds, would entail service cuts across all modes and increased route eliminations. Additionally, it would drastically reduce contracted work like clean-teams and security officers and reduce preventative maintenance. 
+DART Staff calls this scenario as a “High risk to ridership and passenger revenue” 
+All proposed caps or cuts would result in DART that is less reliable and less able to address some of the concerns that have sparked calls for budget cuts. It simply does not make sense to expect increased security, frequency, and service while also restricting DART’s ability to pay for these services. 
+
+
+## Use Below Email Template to contact your DART board member:
+
+-----------------------------------------------------------------------
```
Comment (Schobbish) [inline _posts/fy25-dart-budget-proposal-response.md]: remove hr

**Review by Schobbish — CHANGES_REQUESTED** [review]

**Review by Schobbish — APPROVED** [review]

**Pull request merged**